cmd/util.go

```diff
@@ -384,6 +385,22 @@ func getPodOverrideLabelReason(result *Result, overrideLabel string) (bool, stri
 	if reason := result.Labels[podOverrideLabel]; reason != "" {
 		return true, reason
 	}
+	if rootConfig.kubeauditConfig != "" {
+		var kubeauditConfig = &KubeauditConfig{}
+
+		data, _ := ioutil.ReadFile(rootConfig.kubeauditConfig)
+		yaml.Unmarshal(data, kubeauditConfig)
```

Comment:
iirc this returns an error

Reply:
It does but effectively this err is unreachable, we check if Unmarshaling is fine in root init, if it isn't kubeaudit crashes anyways